src/codegen/mod.rs

```diff
@@ -67,6 +74,11 @@ pub fn generate_mod_rs(env: &Env, root_path: &Path, mod_rs: &[String], traits: &
         writeln!(w, "#[doc(hidden)]")?;
         writeln!(w, "pub mod traits {{")?;
         general::write_vec(w, traits)?;
+        writeln!(w, "}}")?;
+
+        writeln!(w, "#[doc(hidden)]")?;
+        writeln!(w, "pub mod builders {{")?;
+        general::write_vec(w, builders)?;
```

Comment:
Can you check if `builders` is empty before adding this new module? (Would be nice to do it for traits too :smile: ).